### PR TITLE
update gradle build to include distributed tracing samples

### PR DESCRIPTION
**Pull request by spencerhank**

### CODE REVIEW

**Comment by spencerhank** [comment]
@TamimiGitHub - Do yo mind taking a look at this pr when you get a chance?

**Comment by TamimiGitHub** [comment]
Hey @spencerhank - chatted with Aaron about this and we would recommend shortening the names of the samplers. Can you change the name to `dtQueuePublisher` and `dtQueueSubscriber`? Otherwise looks good i'll approve and merge after your changes

File: build.gradle [hunk 1]
```diff
@@ -121,6 +121,8 @@ def scripts = [
     'basicReplier':'com.solace.samples.jcsmp.features.BasicReplier',
     'confirmedPublish':'com.solace.samples.jcsmp.features.ConfirmedPublish',
     'topicToQueueMapping':'com.solace.samples.jcsmp.features.TopicToQueueMapping',
+    'distributedTracingDirectPublisherWithManualInstrumentation':'com/solace/samples/jcsmp/features/distributedtracing/DirectPublisherWithManualInstrumentation',
```
Comment (spencerhank) [inline build.gradle:124]:
```suggestion
    'dtDirectPublisher':'com/solace/samples/jcsmp/features/distributedtracing/DirectPublisherWithManualInstrumentation',
```

File: build.gradle [hunk 2]
```diff
@@ -121,6 +121,8 @@ def scripts = [
     'basicReplier':'com.solace.samples.jcsmp.features.BasicReplier',
     'confirmedPublish':'com.solace.samples.jcsmp.features.ConfirmedPublish',
     'topicToQueueMapping':'com.solace.samples.jcsmp.features.TopicToQueueMapping',
+    'dtDirectPublisher':'com/solace/samples/jcsmp/features/distributedtracing/DirectPublisherWithManualInstrumentation',
+    'distributedTracingDirectSubscriberWithManualInstrumentation':'com/solace/samples/jcsmp/features/distributedtracing/DirectSubscriberWithManualInstrumentation'
```
Comment (spencerhank) [inline build.gradle:125]:
```suggestion
    'dtDirectSubscriber':'com/solace/samples/jcsmp/features/distributedtracing/DirectSubscriberWithManualInstrumentation'
```

**Pull request merged**